### ISSUE TITLE
admin interface to clear failed jobs?

### ISSUE THREAD

**Issue by ccurvey** [rq/django-rq#465]
Am I missing something? 

I have some queues with lots of very old failed jobs.  I'd like to clean things up, but the only "bulk" options I have are to clear the entire queue (which would include queued jobs as well), or "requeue" jobs, which I also don't want to do.

I guess I could go through and delete from the failed list 50 jobs at a time, but that seems to be a lot of pointing and clicking.

(hmm, maybe this was added to 2.4?  I'm still on 2.3.2...)

**Comment by dfrank8** [comment]
Yeah @ccurvey I see this as well. I too have to delete jobs in batches 😆

**Comment by jbarham** [comment]
I had this exact problem so I created a Django management command to do bulk deletions of failed jobs. The code gist is [here](https://gist.github.com/jbarham/de89f85e900bf5f3adce3f0e4d65c5d9).

**Comment by majonathany** [comment]
Hm, perhaps I am missing something but when I go to /django-rq as a superuser I can delete jobs. I am using Django, Django Rest Framework, and Django admin

**Comment by jbarham** [comment]
Sure you can delete 50 jobs at a time from the web UI. But if you have hundreds or thousands of failed jobs to delete that becomes very tedious.

**Comment by selwin** [comment]
Ideally, there's also an option to delete all and not just the 50 selected jobs from the queue, similar to how Django's admin behaves. I'd be happy to accept a PR for this.

![CleanShot 2021-03-31 at 08 02 08@2x](https://user-images.githubusercontent.com/188658/113075600-7329c900-91f7-11eb-8fe5-e48f18e4f9d1.jpg)
